galician/dixital/dixital.md

```diff
@@ -0,0 +1 @@
+Polas rañuras da porta alumeaman alternativamente raiolas de luz artificial, as veces vermellas, outras azuis. A viaxe espacial a través do canal dixital comezara, a información recollida dos cúmulos estelares iniciaba unha nova era.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.